src/Compilers/CSharp/Portable/Declarations/MergedTypeDeclaration.cs

```diff
@@ -142,6 +142,20 @@ public bool HasAwaitExpressions
             }
         }
 
+        public bool IsIterator
+        {
+            get
+            {
+                foreach (var decl in this.Declarations)
+                {
+                    if (decl.IsIterator)
+                        return true;
+                }
+
+                return false;
+            }
```

Comment:
`=> Declarations.Any(d => d.IsIterator);`

Reply:
> => Declarations.Any(d => d.IsIterator);

Follow the established pattern with implementation for properties like that.

---
In reply to: [384789323](https://github.com/dotnet/roslyn/pull/41926#discussion_r384789323) [](ancestors = 384789323)